Update issue templates (#1123)
dotnet/vscode-dotnet-runtime: .github/ISSUE_TEMPLATE/🐜bug-report--vs-code-runtime-extension.md

---

> name: "\U0001F41Cbug report: vs-code runtime extension"

Reviewer: I love the idea of using the bug emoji! I think that this pattern of keeping track of the assignees won't be necessary as it is in the GH UI, but feel free to put that bug elsewhere 😄

Author: Sure😄

about: 'bug report: vs-code runtime extension'
title: ''
labels: ''
assignees: ''

---

> <!--

Reviewer: In markdown, this becomes a comment, so it is not visible.  I reccommend using the quote and emoji technique :)

such as 
> :exclamation: Privacy alert! Blah

Which you can see by trying to edit my comment, it uses the > symbol and then the emoji name in :

Privacy Alert! The contents copied to your clipboard may contain personal data. Prior to posting to GitHub, please remove any personal data which should not be publicly viewable. https://privacy.microsoft.com/en-US/privacystatement`;
-->

**Describe the bug**

> The issue text was copied to the clipboard.  Please paste it into this window.

Reviewer: ```suggestion
Errors the extension may have encountered have been added to your clipboard. Please paste it here and describe what happened and the problem you are facing.
```

> Please attach the log file located at ${ context.logger.getFileLocation() }. Note that this file may contain personal data.

Reviewer: ```suggestion
In addition, please attach the log file located at ${ context.logger.getFileLocation() }. Note that this file may contain personal data.
```

> Please attach the log file located at ${ context.logger.getFileLocation() }. Note that this file may contain personal data.

Reviewer: How does the context log file location get filled out in this situation?

Author: I haven't tried yet but it might be from the `IssueReport.ts`

Author: Tried a couple of hours and appending might not be a solution. Maybe provide a fix path

> **To Reproduce**

Reviewer: ```suggestion
**Reproduction Steps**
```

**Expected behavior**

> **Exceptions (if any)**

Reviewer: ```suggestion
**Exceptions & Errors (if any)**
```

**Further technical details**

> - The IDE (VS / VS Code/ VS4Mac) you're running on, and its version

Reviewer: ```suggestion
- Your VSCode Version,
- The version of the extension that's on your machine.
- Your operating system details.
```
